packages/dashboard/src/components/Card/GridList/GridList.styles.ts

```diff
@@ -17,7 +17,11 @@ export const getStyles = (props: ICustomCssForCells): IGridListStyles => {
     text: {
       fontSize: '14px',
       lineHeight: '16px',
-      color: props.textColor ? props.textColor : '#333333'
+      color: props.textColor ? props.textColor : '#333333',
+      display: 'block',
+      textOverflow: 'ellipsis',
+      overflow: 'hidden',
+      whiteSpace: 'nowrap'
     }
```

Comment:
Do we have tooltip?

Reply:
Kartik is going to fix this in a separate PR

---
In reply to: [215341307](https://github.com/OfficeDev/office-ui-fabric-react/pull/6226#discussion_r215341307) [](ancestors = 215341307)